m4_sessions/automation/session_one/seven_stephanie_robbie_david_annastasia.md

```diff
@@ -1,3 +1,62 @@
+
+*   ["Machines for Thinking"](http://www.economist.com/news/books-and-arts/21669597-computers-will-get-smarter-humans-charge-machines-thinking) (_The Economist_: October 1, 2015)
+*   "The Rise of the Robots by Martin Ford / Humans Need Not Apply by Jerry Kaplan – review"
+https://www.theguardian.com/books/2015/oct/01/the-rise-of-robots-humans-need-not-apply-review (_The Guardian_: October 1, 2015)
+*   ["Jerry Kaplan: Humans Need Not Apply at Google"](https://www.youtube.com/watch?v=JiiP5ROnzw8) (Google: November 12, 2015)
+* ["Yes, the Robots Will Steal Our Jobs, and That's Fine](https://www.washingtonpost.com/posteverything/wp/2016/02/17/yes-the-robots-will-steal-our-jobs-and-thats-fine/?utm_term=.b200c49efca2)
+
+### Groupings
+
+*   What should the group size be for discussions? (Recommend 3)
+*   How should groups be set up? (Recommend some meaningful grouping)
+
+
+### Logistics
+
+| Set Up | Materials Needed |
+| ------ | ---------------- |
+| Students should start the session where ever they want. | Participants need notebooks, pens. |  |
+
+
+## Session Outline
+
+| Timing        | Facilitator Talking Points           | Participant/Facilitator Actions  |
+| ------------- | ------------- | ----- |
+| 2 min | Introduction: Welcome participants.  Introduce ourselves.  Review the outcomes for the session and the connection to the greater Turing mission.  |
+| 8 min | Warm Up: Each student will be insstructed to get a computer to multiply 2 ^ 64 (1 min) and 1min to code a box into existence. Students will then organize in the room along a spectrum of their attitudes toward automation. Describe one end as representing people who feel strongly that automation is, as Elon Musk says, "summoning a demon." On the other end, people who feel automation is going to take humanity to new, positive, exciting potential. (For students in the middle, at least one must choose the opposite view point - devil’s advocate (without being a troll).) (Stephanie will facilitate, Robbie will demonstrate his box) | Facilitator will instruct students to line up in between these poles as appropriate. At facilitator's prompting, students should line up along the spectrum of attitudes toward automation. |
+| 2 min | Introduce video and reflection questions to students. <br> 1) How serious an impact do you think automation will have on our economy from now until you retire? <br> 2) Where do you think you fall across the spectrum of people that will be affected by these changes? <br> 3) What do you think it means to be a responsible citizen around this issue? <br> 4) What role can Turing play in advancing civic responsibility in response to the challenges automation will bring?  <br> | Anna will facilitate. Students get out paper and pen to prepare for video. |
+| 15 min | Kick off ["Humans Need Not Apply"](https://www.youtube.com/watch?v=7Pq-S557XQU) video. | Students should take notes during the video.  Ask them to put away cell phones/computers if needed. |
```

Comment:
if this is in the pre-reading session, then do you want to show it during session time?